roles/openshift_monitor_availability/defaults/main.yml

```diff
@@ -14,3 +14,6 @@ openshift_monitor_app_create_image: "{{ openshift_monitor_app_create_image_prefi
 openshift_monitor_app_create_run_interval: 5m
 openshift_monitor_app_create_timeout: 5m
 openshift_monitor_app_create_log_level: 0
+
+openshift_web_console_smoke_test_image: docker.io/benjaminapetersen/web-console-smoke-test:latest
```

Comment:
This should follow the pattern for app create image vars above. We should probably use a similar names for these tests throughout.

Reply:
Yeah so I wans' really sure if thats needed since both `openshift_deployment_type` variable types are pointing to the same image.
> We should probably use a similar names for these tests throughout.
You mean something like `openshift_monitor_web_console_image` ?